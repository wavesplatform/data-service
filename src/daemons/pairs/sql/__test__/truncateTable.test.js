const truncateTable = require('../truncateTable');

describe('pairs daemon sql test', () => {
  it('truncate table', () => {
    expect(truncateTable('pairs')).toMatchSnapshot();
  });
});
