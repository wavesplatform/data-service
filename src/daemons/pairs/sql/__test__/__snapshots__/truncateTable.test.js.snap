// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pairs daemon sql test truncate table 1`] = `"truncate \\"pairs\\" restart identity"`;
